Source/HelixToolkit.Wpf.SharpDX/Utilities/ObjReader.cs

```diff
@@ -1180,13 +1180,13 @@ private Material CreateMaterial(string texturePath)
                     //AmbientMap = this.AmbientMap,
 
                     DiffuseColor = this.Diffuse,
-                    DiffuseMap = (this.DiffuseMap == null) ? null : LoadImage(this.DiffuseMap),
+                    DiffuseMap = (this.DiffuseMap == null) ? null : LoadImage(Path.Combine(texturePath, this.DiffuseMap)),

```

Comment:
@deviloper Just add a commit to your branch and push it to your repo. GitHub will automatically see the new commit and include it in the PR.